### DISCUSSION TITLE
API // PUT Products only Values being changed.

### DISCUSSION THREAD

**Discussion by chrisBradley-dev** [unopim/unopim#86] [Ideas]
Hey, 

I would like to suggest revising the API so that you only need to pass the values that you need to update in a PUT and use the SKU as the identifier. 

Currently, to update unoPIM you would need to GET the product, change the values you need to change then send the payload back. 

For example, if you needed to PUT inventoryQuantity onto a Product, you would need to GET the product, then change the received payload with the new InventoryQuantity then return the whole payload. (same goes for other things, this is just an example).

**Comment by navneetkumar-pim-webkul** [comment]
Hi @chris-chameleonbrands,

Thank you for the suggestion! We will review this and work on providing an API that allows patching the product using the PATCH method.

Your feedback is greatly appreciated, and we’ll keep you updated on the progress.

Best regards,
